### ISSUE TITLE
Focal Tversky Loss and Attention Gated U-Net

### ISSUE THREAD

**Issue by jchen42703** [jchen42703/kits19-cnn#6]
How about we get an implementation of this up and running? Here's some source code:
* https://github.com/nabsabraham/focal-tversky-unet (keras) [2D]
* https://github.com/ozan-oktay/Attention-Gated-Networks (pytorch) [3D]

**Comment by jchen42703** [comment]
Doesn't work that well and hard to tune.
